sale_financial_risk_info/models/sale.py

```diff
@@ -9,10 +9,6 @@ class SaleOrder(models.Model):
     _inherit = "sale.order"
 
     risk_info = fields.Html(compute="_compute_risk_info")
-    risk_remaining_value = fields.Monetary(related="partner_id.risk_remaining_value")
```

Comment:
This shouldn't be removed. You didn't rebase properly.

Reply:
I have rebased and I can't see in 15.0 the `risk_remaining_value` field for the res.partner model in the `sale_financial_risk` module and therefore I am getting an error having a related partner_id.risk_remaining_value field in sale.order in this (sale_financial_risk_**info**) module. But also not even in v14 that related is used in this module (nothing is shown or done with it)

Reply:
This module is not in v15. And in v14, it's here: https://github.com/OCA/credit-control/blob/771640902939557782bd87a4a6efacd0d6fc5351/sale_financial_risk_info/models/sale.py#L12

The problem is that your migration commit is removing these fields.

Reply:
I know what happens. I did research and the PR that adds the field was done in v14.0 after the migration to v15 of the account_financial_risk module, I did a forward-port of that PR to recover those fields in res.partner. 
PR: https://github.com/OCA/credit-control/pull/239